scvi/model/base/_pyromixin.py

```diff
@@ -46,6 +46,41 @@ def on_train_start(self, trainer, pl_module):
             break
 
 
+class PyroModelGuideWarmup(Callback):
```

Comment:
Which test would fail if this doesn't work?

Reply:
low level test